uniswap/model/sys_params.py

```diff
@@ -3,5 +3,7 @@
 sys_params = {
     'fee_numerator': [997],
     'fee_denominator': [1000],
-    'uniswap_events': [pd.read_pickle('~/Documents/uniswap/uniswap_events.pickle')]
+    'uniswap_events': [pd.read_pickle('~/Documents/uniswap/uniswap_events.pickle')],
```

Comment:
It'd be better to use a relative path here.

Reply:
Agreed